src/dagnode-stream.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const TransformStream = require('readable-stream').Transform
+const streamToValue = require('./stream-to-value')
+const getDagNode = require('./get-dagnode')
+
+/*
+  Transforms a stream of objects to DAGNodes and outputs them as objects.
+
+  Usage: inputStream.pipe(DAGNodeStream({ send: send }))
+
+  Input object format:
+  {
+    Name: '/path/to/file/foo.txt',
+    Hash: 'Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP'
+  }
+
+  Output object format:
+  {
+    path: '/path/to/file/foo.txt',
+    hash: 'Qma4hjFTnCasJ8PVp3mZbZK5g2vGDT4LByLJ7m8ciyRFZP',
```

Comment:
@diasdavid I'm not sure about the `multihash` vs. `hash`. It's a breaking change again and I'd be hesitant to change it in this PR. Have we made the change elsewhere in the code already? Does anything that returns DAGNode-like information return .multihash instead of .hash? If we have, I can change it so that we're consistent, but if not, I would leave it for later.